llvm/lib/Target/X86/X86.td

```diff
@@ -1139,6 +1139,28 @@ def ProcessorFeatures {
   list<SubtargetFeature> GNRDFeatures =
     !listconcat(GNRFeatures, GNRDAdditionalFeatures);
 
+  // Diamondrapids
```

Comment:
(trivial) Diamondrapids -> DiamondRapids (for consistency)

Reply:
[52e63a0](https://github.com/llvm/llvm-project/pull/113881/commits/52e63a0f4b48ad106471343f2f48976bd9b84661)